bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/GhostAsterism.scala

```diff
@@ -33,11 +33,11 @@ object GhostAsterism {
     * Guide state can also be explicitly disabled, say, in a crowded field where
     * guide fibers are less effective.
     */
-  sealed trait GuideFiberState extends Product with Serializable
+  sealed abstract class GuideFiberState private (val name: String) extends Product with Serializable
```

Comment:
I don't see another way for a sealed type to be used with `ParamCodec`, and followed the example used for `MagnitudeBand` in:

https://github.com/gemini-hlsw/ocs/blob/develop/bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/TargetParamCodecs.scala

https://github.com/gemini-hlsw/ocs/blob/develop/bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/MagnitudeBand.scala